SharpBucket/V2/ISharpBucketRequesterV2.cs

```diff
@@ -0,0 +1,35 @@
+﻿using SharpBucket.V2.EndPoints;
+
+namespace SharpBucket.V2
+{
+    /// <summary>
+    /// Interface that expose all the methods that an end point V2 could use to perform calls to the BitBucket API.
+    /// </summary>
+    public interface ISharpBucketRequesterV2 : ISharpBucketRequester
+    {
+        /// <summary>
+        /// Get the Teams end pPoint.
+        ///  Give access to the data relative to the teams.
+        /// </summary>
+        TeamsEndPoint TeamsEndPoint();
```

Comment:
I wonder if we should expose interfaces for the end points instead of concrete types
ITeamsEndPoint instead of TeamsEndPoint

Reply:
In fact exposing interfaces would generate breaking changes on that interfaces each time we add new methods (which happens each time we cover a bit more the API V2)

While having concrete types allow us to write new methods in them without generating a breaking change (since the implementation of the new methods is provided by the concrete type)